woodwork/statistics_utils/_get_dependence_dict.py

```diff
@@ -103,24 +103,33 @@ def _get_dependence_dict(
     start_time = timer()
     if target_col is not None and target_col not in list(dataframe.columns):
         raise ValueError("target_col '{}' not in the dataframe".format(target_col))
-
+    boolean_columns = dataframe.ww.select(["Boolean", "BooleanNullable"]).columns.values
+    bool_to_int = {col: "IntegerNullable" for col in boolean_columns}
+    dataframe_with_bools_to_int = dataframe.ww.copy()
+    dataframe_with_bools_to_int.ww.set_types(bool_to_int)
     returned_measures, calc_order, calc_max = _parse_measures(measures)
 
     unit = "calculations"
 
     # get valid columns for dependence calculations
     if "pearson" in calc_order:
         pearson_types = get_valid_pearson_types()
-        pearson_columns = _get_valid_columns(dataframe, pearson_types)
+        pearson_columns = _get_valid_columns(dataframe_with_bools_to_int, pearson_types)
```

Comment:
Are the dependence calculations only being done on bool columns now?

Reply:
It runs it on all columns! The boolean columns are just now being set properly for calculations.